recipe/meta.yaml

```diff
@@ -2,70 +2,109 @@
+    # Use `openssl sha256 file.tar.gz` to get sha256 stamp
+    url: https://github.com/openfast/openfast/archive/v{{ version }}.tar.gz
+    sha256: 791c1098f7d3df33bb3eeb73043badb6987c9014a91ff7547613673429bd380e
+    #git_url: https://github.com/openfast/openfast.git
+    #git_rev: v{{ version }}
+    #git_depth: 1 # (Defaults to -1/not shallow)
+    #patches:
+    #  - fPIC.patch
 
 build:
-  number: 0
-  missing_dso_whitelist:
-    - "*/libopenfastlib.dll"   # [win]
-requirements:
-  build:
-    - {{ stdlib("c") }}                 # [not win]
-    - {{ compiler('c') }}               # [not win]
-    - {{ compiler('cxx') }}             # [not win]
-    - {{ compiler('fortran') }}         # [not win]
-    - {{ compiler('m2w64_c') }}         # [win]
-    - {{ compiler('m2w64_cxx') }}       # [win]
-    - {{ compiler('m2w64_fortran') }}   # [win]
-    - cmake
-    - make         # [not win]
-    - ninja        # [win]
-  host:
-    - libgomp      # [not osx]
-    - llvm-openmp  # [osx]
-    - openblas     # [not win]
-    - libblas      # [win]
-    - liblapack    # [win]
-    #- libcblas
-    #- liblapacke    
-  run:
-    - libgomp      # [not osx]
-    - llvm-openmp  # [osx]
+    number: 1
+        
+outputs:
+    - name: openfast
+      version: {{ version }}
+      script: build.sh                     # [not win]
+      script: bld.bat                      # [win]
+      build:
+          run_exports:
+              - {{ pin_subpackage('openfast', max_pin='x.x') }}
+          missing_dso_whitelist:
+              - "*/libopenfastlib.dll"   # [win]
+      requirements:
+          build:
+              - {{ stdlib("c") }}                 # [not win]
+              - {{ compiler('c') }}               # [not win]
+              - {{ compiler('cxx') }}             # [not win]
+              - {{ compiler('fortran') }}         # [not win]
+              - {{ compiler('m2w64_c') }}         # [win]
+              - {{ compiler('m2w64_cxx') }}       # [win]
+              - {{ compiler('m2w64_fortran') }}   # [win]
+              - cmake
+              - make         # [not win]
+              - ninja        # [win]
+          host:
+              - libgomp      # [not osx]
+              - llvm-openmp  # [osx]
+              - openblas     # [not win]
+              - libblas      # [win]
+              - liblapack    # [win]
+              #- libcblas
+              #- liblapacke    
+          run:
+              - libgomp      # [not osx]
+              - llvm-openmp  # [osx]
 
-test:
-  commands:
-    - openfast -h
-    - FAST.Farm -h
-    - aerodyn_driver -h
-    - hydrodyn_driver -h
-    - moordyn_driver -h
-    - servodyn_driver -h
-    - turbsim -h
+      test:
+          commands:
+              - openfast -h
+              - FAST.Farm -h
+              - aerodyn_driver -h
+              - hydrodyn_driver -h
+              - moordyn_driver -h
+              - servodyn_driver -h
+              - turbsim -h
```

Comment:
We can add some checks on other modules:
- aerodisk_driver
- sed_driver
- inflowwind_driver
- seastate_driver

I don't think most of these codes actually do anything with the `-h` flag.  But they should return something.

Reply:
I checked on the exit codes. With the `-h` flag, they all return an exit code of `0`, but some give a message of `Aborting`.  So conda should be ok since it only checks the exit code

Reply:
Also
- beamdyn_driver -h
- subdyn_driver -h

Reply:
There are a couple of other driver codes, but I don't think they are commonly used or compiled so probably not worth checking. (Not sure which are built for conda right now).